src/components/Private/Private.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import MainProvider from '../../state/MainProvider';
+import AuthProvider from '../../providers/Auth';
+
+import Private from './Private.component';
+
+describe('Private tests', () => {
+  test('check correct render of login', async () => {
+    const wrapper = ({ children }) => {
+      return (
+        <MainProvider>
+          <AuthProvider>
```

Comment:
Not sure if I already shared this with you, but you can have a custom render for your tests where you can add all the providers needed for your components. Check the following documentation and let me know if you have any questions:

https://testing-library.com/docs/react-testing-library/setup/#custom-render